modules/nw-multus-configure-dualstack-ip-address.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/configuring-additional-network.adoc
+
+:_content-type: PROCEDURE 
+
+[id="nw-multus-configure-dualstack-ip-address_{context}"]
+= Creating a configuration for assignment of dual-stack IP addresses dynamically
+
+Dual-stack IP address assignment can be configured with the `ipRanges` parameter for:
+* IPv4 addresses
+* IPv6 addresses
+* multiple IP address assignment 
+
+.Procedure
+
+. Set type to whereabouts.
+. Use `ipRanges` to allocate IP addresses.
+
```

Comment:
You need a + sign here to indent the example.

Reply:
Thank you